tmt/tests/test-00-bootc-install.sh

```diff
@@ -36,6 +36,23 @@ if [ "$TMT_REBOOT_COUNT" -eq 0 ]; then
     ls -al "$BOOTC_TEMPDIR"
 
     pushd "$TMT_TREE"
+        find /etc/yum.repos.d/*.repo -name tag-repository.repo | xargs -I '{}' cp -v '{}' repos.d/
+
+        if [ -e /etc/yum.repos.d/test-artifacts.repo ]; then
+            mkdir -p rpms
+            repoquery --disablerepo=\* --enablerepo=test-artifacts --qf "%{SOURCERPM} %{NAME}" > T-A-pkgs.pkglist
```

Comment:
Please write it as <code>dnf repoquery --repo=test-artifacts</code>:

Bare repoquery program won't work with dnf5.
The --disablerepo --enablerepo can be simplified to --repo.